examples/server/server.c

```diff
@@ -2824,7 +2824,11 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
 #ifdef WOLFSSL_SNIFFER
     if (cipherList == NULL && version < 4) {
         /* static RSA or static ECC cipher suites */
-        const char* staticCipherList = "AES128-SHA:ECDH-ECDSA-AES128-SHA";
+        const char* staticCipherList = "AES128-SHA:ECDH-ECDSA-AES128-SHA"
+        #if defined(HAVE_ARIA)
```

Comment:
Perhaps it would be better to supply the cipher list for these test cases instead of forcing it here for the sniffer?